### PR TITLE
Add information about mungegithub deployment

### PR DESCRIPTION
**Pull request by crimsonfaith91**
Addressing https://github.com/kubernetes/test-infra/issues/3215

### CODE REVIEW

**Comment by apelisse** [comment]
@grodrigues3 please take a quick look

File: mungegithub/README.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ TARGET=kubernetes REPO=docker.io/$USERNAME APP=submit-queue KUBECONFIG=/path/to/
 ```
 
 **Note** The submit-queue should always be deployed from master so that look at
```
Comment (grodrigues3) [inline mungegithub/README.md]: nit: so that `looking` at...

File: mungegithub/README.md [hunk 2]
```diff
@@ -39,7 +39,8 @@ TARGET=kubernetes REPO=docker.io/$USERNAME APP=submit-queue KUBECONFIG=/path/to/
...
-the master branch (configmap, etc) reflects what is currently running.
+the master branch (configmap, etc) reflects what is currently running. The deployment may require
+checkout based on commit hash found in [Godeps.json](https://github.com/kubernetes/test-infra/blob/master/Godeps/Godeps.json).
```
Comment (grodrigues3) [inline mungegithub/README.md]: This isn't super clear to me.  I should go into the godeps file, find the appropriate hash and checkout all of the appropriate commits from everyone of the repos.

Comment (apelisse) [inline mungegithub/README.md]: Yes, that's called `godep restore` ;-)

Comment (crimsonfaith91) [inline mungegithub/README.md]: Let me summarize the steps and introduce usage of `godep restore`.

**Comment by crimsonfaith91** [comment]
addressed!

**Comment by apelisse** [comment]
This is a bit too convoluted IMO. What about: "Make sure you have the proper version of each dependencies by running `godep restore`."

**Comment by crimsonfaith91** [comment]
Rephrased in such a way that `godep restore` is mentioned before explaining why it is required to run the command.

**Comment by apelisse** [comment]
I think this is even more convoluted. I doubt this is a good place to describe what `godeps restore` does.

**Comment by crimsonfaith91** [comment]
that's why i include it in a bracket :)

**Comment by grodrigues3** [comment]
Can we link to the godep restore description header? 

https://github.com/tools/godep#restore

**Comment by crimsonfaith91** [comment]
ok. that would be helpful.

**Comment by crimsonfaith91** [comment]
addressed!

**Pull request merged**